src/cli.toit

```diff
@@ -75,6 +103,10 @@ class Command:
   The $help is a longer description of the command that can span multiple lines. Use
     indented lines to continue paragraphs (just like toitdoc). The first paragraph of the
     $help is used as short help, and should have meaningful content on its own.
+
+  The $run callback is invoked when the command is executed. It is given the $App and the
```

Comment:
I think I agree with the chosen order here (app before parsed), but it would probably have caused less churn to add the app as the last parameter.

Reply:
agreed.
I think it's relatively easy to fix, though. So hopefully it won't really be a huge issue.